OpenRA.Mods.Common/Traits/World/EditorActorLayer.cs

```
@@ -106,7 +106,8 @@ EditorActorPreview Add(string id, ActorReference reference, bool initialSetup =
 
 			var preview = new EditorActorPreview(worldRenderer, id, reference, owner);
 			previews.Add(preview);
-			screenMap.Add(preview, preview.Bounds);
+			if (!preview.Bounds.IsEmpty)
```

Please throw a sensible exception here instead of silently ignoring it.